app/models/site.rb

```diff
@@ -50,7 +50,7 @@ def select_proxy(older_than=-1)
         Proxy::NoProxy
       elsif proxy_ts > threshold_ts
         # The proxy we found was too recently used.
-        Proxy::NoColdProxy.new(proxy_ts - threshold_ts)
+        Proxy::NotReady.new(proxy_ts, threshold_ts, proxy_id)
```

Comment:
If I touch it here then it will break the v1 API.  I'm touching it in the [client code instead](https://github.com/Raybeam/zartan/blob/d57725ff2d7b1de5e60dbd0d118a69140ddb09db/lib/client.rb#L35).